src/components/Book.jsx

```diff
@@ -1,24 +1,31 @@
+import axios from 'axios';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
+import { baseUrl } from '../redux/store';
 
 function Book(props) {
   const dispatch = useDispatch();
 
+  const handleRemove = async (id) => {
+    await axios.delete(baseUrl + id)
+      .then(() => dispatch(removeBook(id)));
+  };
+
```

Comment:
- [ ] Same here, Please use createAsyncThunk for creating thunk remove action creator, and kindly move it to the books redux slice and import and use it here in `handleRemove` function.